src/connectors/dispatcher/impl.js

```diff
@@ -43,5 +46,65 @@ module.exports = new class extends Connector {
         return result;
     }
 
+    async fetchPlaybookRuns (filter = null, fields = null, sort_by = null) {
+        const uri = new URI(config.platformHostname);
+        uri.path('/api/playbook-dispatcher/v1/runs');
+
+        if (filter) {
+            uri.addQuery('filter', filter);
```

Comment:
both `filter` and `fields` use LHS bracket syntax (e.g. `filter[labels][service]=remediations`). I don't think you'll be able to specify these like this. I think you'll need to do something like `uri.addQuery('filter[labels][service]', 'remediations')`

Based on https://github.com/medialize/URI.js/issues/86 it looks like urijs does not support this natively. Perhaps a custom encoder could be written, though.

Reply:
@jharting would it also be possible to create a JSON object like what is shown in the API docs and pass that as the data for the filter parameter?  

i.e.
```
{
  "run": {
    "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
    "labels": {
      "additionalProp1": "string",
      "additionalProp2": "string",
      "additionalProp3": "string"
    },
    "service": "string"
  },
  "status": "running"
}
```

Reply:
Based on what I found it looks like urijs does not natively support deepObject query params i.e. you'll need to add logic for converting it.

Reply:
@jharting I think I found a solution in the `qs` library.  I'll have to do some additional manipulation when I declare the filter and fields variable for each call, but this looks like it will return the correct format to be encoded by URI.js.